flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/utils/PlannerMocks.java

```diff
@@ -134,7 +135,15 @@ public PlannerContext getPlannerContext() {
 
     public PlannerMocks registerTemporaryTable(String tableName, Schema tableSchema) {
         final CatalogTable table =
-                CatalogTable.of(tableSchema, null, Collections.emptyList(), Collections.emptyMap());
+                CatalogTable.of(
+                        tableSchema,
+                        null,
+                        Collections.emptyList(),
+                        Map.of(
+                                "connector",
+                                TestSimpleDynamicTableSourceFactory.IDENTIFIER(),
+                                "bounded",
```

Comment:
Why 'bounded' should always be 'true?'

Reply:
In reality, true and false are indistinguishable and currently not utilized; they simply function as placeholder fields for TestTableSource. I will set the 'bounded' parameter as optional to avoid any potential misunderstandings.